.github/workflows/buid_and_release.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build_and_release:
     runs-on: ubuntu-latest
-    container: obolibrary/odkfull:v1.4.1
+    container: obolibrary/odkfull:latest
```

Comment:
## The fix
This was the problem. This older version of the production ODK container was not up-to-date enough.

### Advantages to `latest` > fixed version
- Have access to the latest features and fixes.
- Don't have to worry about checking that the version is up-to-date / correct whenever work is done on the repo.

### Disadvantages
- Can introduce breaking changes _(it's because of this that I chose a fixed version before)_

### Alternatives? (probably not)
It could be nice to use a wildcard `*` on the minor or patch version, but not the major version. However, upon some googling, generative AI suggested I can do this, but [search results](https://emby.media/community/index.php?/topic/127712-docker-tags-majorminor-wildcard/) (and my terminal) show that I cannot. That makes sense. I think container tags are only string labels; doesn't support a proper versioning system like semver.

Reply:
I personally prefer fixed versions to minimise churn from breaking changes. These happen more frequently as python code ages..

But I dont mind strongly in this case.

Reply:
(Note that we may not be monitoring dozens of repos with actions which we rely on to supply as with up to date data. If one of them stops working, maybe we would never know).

Reply:
After I merged this, I thought about it later and I think I agree with you. I made a new PR and fixed it to 1.5:
- #25

> If one of them stops working, maybe we would never know).

@matentzn Just to clarify. Are you saying that this is a risk of setting the version to `latest`? And, additionally, are you just not sure if (the appropriate) people are receiving (and actually taking seriously / reviewing) notifications for GH action failures?
For me, I see them as an email and I always review, but IDK about others.

Reply:
> And, additionally, are you just not sure if (the appropriate) people are receiving (and actually taking seriously / reviewing) notifications for GH action failures?

I can tell you for myself that I receive hundreds of notifications daily and I will miss many; and I know that compared to many people I know that are even more busy that they will be guaranteed to miss theirs.

Reply:
> . Are you saying that this is a risk of setting the version to latest?

Using a versioned container should minise the risk of failure, as no upstream tool changes can impact the execution.